### PR TITLE
EW-355

### PR DESCRIPTION
**Pull request by SimoneRadtke-Cap**
# Description

An ErWIn project website for public relations.

## Links to Tickets or other pull requests

- https://ticketsystem.dbildungscloud.de/browse/EW-355

## Approval for review

- [ ] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "dependencies": {
+    "bootstrap": "^5.2.2"
+  },
+  "devDependencies": {
+    "autoprefixer": "^10.4.13",
+    "nodemon": "^2.0.20",
+    "sass": "^1.56.0",
+    "ncp": "^2.0.0"
```
Comment (psachmann) [inline package.json:9]: This dependency seems very old and unmaintained (last publish 8 years ago).

File: src/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+h1, h2 {
+  font-weight:bolder;
+  color: #9E292B;
+}
+
+.text {
+  font-size: 1.3em;
+  margin: 10px auto;
+}
```
Comment (psachmann) [inline src/css/styles.css]: Why do we need css and sass both of them? Can't we use just one of them?

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#anchor-impressum">Impressum</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+    </nav>
+
+    <main>
```
Comment (psachmann) [inline src/index.html:39]: Maybe we can add an `article` tag for better search engine results?

Comment (SimoneRadtke-Cap) [reply src/index.html:39]: Done

**Review by pkleybolte — APPROVED** [review]

File: src/css/styles.css (hunk 2 above)
Comment (SimoneRadtke-Cap) [reply src/css/styles.css]: Done

File: package.json (hunk 1 above)
Comment (SimoneRadtke-Cap) [reply package.json:9]: We only need the package for the build and have decided that it is sufficient.

**Pull request merged**